articles/exception-handling-in-javascript/index.md

```diff
@@ -0,0 +1,159 @@
+- **Syntax Error**: A Syntax Error exception is raised while compiling a code with syntax errors.
+- **TypeError**: A `TypeError` exception is raised when a value is different from the one expected.
+- **URIError**: A `URIError` exception is raised by `encodeURI()` and `decodeURI()` methods.
+
+### How to Handle Exceptions in JavaScript
+We now know what exceptions are. It's time to understand how to handle them to prevent our programs from crashing. JavaScript handles exceptions in `try-catch-finally` statements and `throw` statements.
+
+#### Key Terms
+* a **`try-catch-finally`** statement is a code or program that handles exceptions.
+* the **`try`** clause identifies the code that generates exceptions.
+* the **`catch`** clause receives exceptions that are thrown.
+* a **`finally`** clause gets executed always.
+* the **`throw`** statement generates exceptions.
+
+#### Throw Statements
+The `throw statement` is used to raise your built-in exceptions. Below is an example of a ```throw``` statement.
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exception Handling</title>
+</head>
+<body>
+    <script type="text/JavaScript">
+       function myFunction() {
+           const x = 50;
+           const y = 0;
+           try{
+               if (y === 0) {
+                   throw("This is division by zero error");   
+               }else{
+                   const z = x / y;
+               }
+           }catch(error){
+               alert("Error: " + error);
+           }
+       }
+    </script>
+    <p>See the result by clicking the button below</p>
+    <button type="button" onclick="myFunction()">Click Here</button>
+</body>
+</html>
+```
+
+#### Try Catch Statements
+The `try` clause has the main code that may generate exceptions. In case an exception is thrown, the `catch` clause is executed.
+
+Here is an example of a `try-catch` statement. 
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Throw Catch Statement</title>
+</head>
+<body>
+    <script type="text/Javascript">
+    function myFunction() {
+        const j = 70;
+        try {
+            alert("The value of j is : " + j);
+        } catch (error) {
+            alert("Error: "+ error.message);
+        }  
+    }
+    </script>
+    <p>Click the button to see the output</p>
+    <button type="button" onclick="myFunction()">Click Here</button>
+</body>
+</html>
+```
+
+In the example above, the `try` clause calls an `alert()` function. The alert() function is not declared therefore an exception will be raised. The exception is placed in the `error` at the `catch` clause and gets executed.
```

Comment:
the `alert` function is a method in javascript hence your code will actually run successfully. Try using a different function name. 

This is a technical issue